Git-commit: 1e52616c9a6899e65187082326e544df96719a16
From: Zheng Zengkai <zhengzengkai@huawei.com>
Date: Sat, 27 Mar 2021 19:22:55 +0800
Subject: [PATCH] arm64: mm: fixes reserve_crashkernel twice by mistake
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.18.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: bugfix
bugzilla: 50423
CVE: NA

------------------------------

reserve_crashkernel() is called twice by mistake, remove the second one.

Fixes: beb42167fd0b ("arm64: mm: Move reserve_crashkernel() into mem_init()")
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Reviewed-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/mm/init.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/mm/init.c b/arch/arm64/mm/init.c
index d60507def2e6..3f09e8b9809e 100644
--- a/arch/arm64/mm/init.c
+++ b/arch/arm64/mm/init.c
@@ -652,12 +652,6 @@ void __init bootmem_init(void)
 	reserve_quick_kexec();
 #endif
 
-	/*
-	 * request_standard_resources() depends on crashkernel's memory being
-	 * reserved, so do it here.
-	 */
-	reserve_crashkernel();
-
 	reserve_pin_memory_res();
 #ifdef CONFIG_ARM64_PMEM_RESERVE
 	reserve_pmem();
-- 
2.31.1

